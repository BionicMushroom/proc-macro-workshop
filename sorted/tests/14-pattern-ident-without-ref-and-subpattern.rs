#[allow(unreachable_patterns)]
#[sorted::check]
fn main() {
    let x = 10;

    #[sorted]
    match x {
        def => println!("{def}"),
        abc => println!("{abc}"),
    }
}
