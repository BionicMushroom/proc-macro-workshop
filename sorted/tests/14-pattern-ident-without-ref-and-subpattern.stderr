error: abc should sort before def
 --> tests/14-pattern-ident-without-ref-and-subpattern.rs:9:9
  |
9 |         abc => println!("{abc}"),
  |         ^^^
